test/image-blueprints/rhel92-microshift-previous-minor.toml

```diff
@@ -0,0 +1,30 @@
+name = "rhel-9.2-microshift-4.${PREVIOUS_MINOR_VERSION}"
+description = ""
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-92"
+
+# FIXME: Temporarily pin the openvswitch version.
+[[packages]]
+name = "openvswitch3.1"
+version = "3.1.0-14.el9fdp"
```

Comment:
This is no longer necessary due to an explicit dependency inside microshift RPM

Reply:
Do we have a published 4.13 build with that dependency in place, yet?

Reply:
It's merged, but I do not know if it's published.

Reply:
OK, I'd rather keep this in place for now then to ensure that the upgrade job has a chance of passing. When we have a real fix, we can remove this. I'll open a ticket to make sure we do it when we branch for 4.15.